docker/worker/Dockerfile

```diff
@@ -66,6 +68,12 @@ RUN cp /home/turbinia/password.lst /root/
 
 RUN pip3 install impacket --no-deps
```

Comment:
Can we please move any package installation above the 'ADD . /tmp/' line in the Dockerfile(s)? Moving it below that line invalidates the layers each time a source file is changed and forces the pip install on each image build.

Reply:
Done